paddle/fluid/operators/activation_op.h

```diff
@@ -101,6 +118,7 @@ struct SigmoidFunctor : public BaseActivationFunctor<T> {
 
 template <typename T>
 struct SigmoidGradFunctor : public BaseActivationFunctor<T> {
+  bool Inplace() const { return true; }
```

Comment:
Can we have this in C++
static char* inplace_acts = ["sigmoid", "exp", ...]
bool IsInPlaceAct(const string& n) {
  return n in inplace_acts;
}
Inplace() { IsInPlaceAct("exp") }

in Python, we expose IsInPlaceAct() as core.is_in_place_act()